server.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const path = require('path');
+
+const apiRouter = require('./api/api.router.js');
+
+const app = express();
+const port = process.env.port || 3000;
+
+app.use('/api', apiRouter);
+app.use(express.static(path.join(__dirname, '/dist/kristina-project')));
+
+app.get('*', (req, res) => {
+    res.status(404).send('Not found');
```

Comment:
Could face an issue directly accessing angular-defined route (ie `http://myblog.com/feed`, where `/feed` is route for angular component), since webserver is sending angular index.html for `/` only